src/base/bittorrent/session.cpp

```diff
@@ -370,12 +370,12 @@ Session::Session(QObject *parent)
     , m_isIPFilteringEnabled(BITTORRENT_SESSION_KEY("IPFilteringEnabled"), false)
     , m_isTrackerFilteringEnabled(BITTORRENT_SESSION_KEY("TrackerFilteringEnabled"), false)
     , m_IPFilterFile(BITTORRENT_SESSION_KEY("IPFilter"))
-    , m_announceToAllTrackers(BITTORRENT_SESSION_KEY("AnnounceToAllTrackers"), false)
+    , m_announceToAllTrackers(BITTORRENT_SESSION_KEY("AnnounceToAllTrackers"), true)
```

Comment:
there was some unsettled discussion in another thread, I don't think there is conscious to change this yet.

Reply:
What are to pros/cons, potential problems, can you link the relevant thread please? I couldn't find it by searching.

Reply:
https://github.com/qbittorrent/qBittorrent/pull/11279

Reply:
Thanks, I read through the issue and came to the following conclusion:
It is better to enable both settings (AnnounceToAllTiers, which is already enabled by default, and this one), since it maximizes peer discoverability, at the cost of a tiny violation of the spec. It follows the principle of least surprise: nowadays users expect all trackers to be contacted always, since the reason they add them in the first place is to find more peers, regardless of the intention and motivation behind the original multi tracker spec. Plus, it is always possible to toggle the setting off to strictly follow the spec. But for the average user, in 2020, I think this is the better default.

Reply:
By the way, I think the one of the main issues with multiple trackers/adding new trackers in qbt is sumarised in this comment: https://github.com/qbittorrent/qBittorrent/pull/11279#issuecomment-536162910

There is no fine-grain control of which tier a tracker should be added to and how many tiers there should be.